torch/_inductor/ir.py

```diff
@@ -1817,6 +1817,44 @@ def as_storage_and_layout(x, freeze=True, want_contiguous=False, stride_order=No
     raise NotImplementedError
 
 
+def is_pointwise_with_channels_last_inputs(x):
+    if isinstance(x, TensorBox):
+        return is_pointwise_with_channels_last_inputs(x.data)
+    if isinstance(x, StorageBox) and isinstance(x.data, Pointwise):
+        all_reads = x.data.get_reads()
```

Comment:
Similar as in https://github.com/pytorch/pytorch/pull/122760, we can get the buffer from `read.name`. Will it be easy to check the buffer's layout?

Reply:
Do I understand correctly that we can share some logic from #122760 too? Both need to get layouts of the input buffers.